corehq/apps/app_manager/fixtures/mobile_ucr.py

```diff
@@ -109,11 +109,15 @@ def should_sync(self, restore_state):
 
     def _get_apps(self, restore_state, restore_user):
         app_aware_sync_app = restore_state.params.app
+        web_apps_restore = restore_state.params.device_id.startswith('WebAppsLogin')
```

Comment:
Looks like there's a util for this:
https://github.com/dimagi/commcare-hq/blob/ae87eb2ebdd1a06437438327ebd02dfa913d6f83/corehq/ex-submodules/casexml/apps/phone/restore.py#L311-L313
```suggestion
        web_apps_restore = restore_state.params.is_webapps
```

Reply:
See e76249bb00aa8db537b28f7e741550d0d8a029be